kymatio/scattering1d/filter_bank.py

```diff
@@ -1,8 +1,7 @@
 import numpy as np
 import math
 import warnings
-from .backend.numpy_backend import ifft1d_c2c as ifft
-
+from scipy.fftpack import ifft
```

Comment:
sure about this? what's the motivation

Reply:
I don't want the filters to be correlated with the backend